src/pages/ReimbursementAccount/ValidationStep.js

```diff
@@ -165,10 +166,12 @@ class ValidationStep extends React.Component {
         }
 
         const maxAttemptsReached = lodashGet(this.props, 'reimbursementAccount.maxAttemptsReached');
+        const isVerifying = !maxAttemptsReached && state === BankAccount.STATE.VERIFYING;
+
         return (
             <View style={[styles.flex1, styles.justifyContentBetween]}>
                 <HeaderWithCloseButton
-                    title={this.props.translate('validationStep.headerTitle')}
+                    title={isVerifying ? this.props.translate('validationStep.letsChatTitle') : this.props.translate('validationStep.headerTitle')}
                     stepCounter={{step: 5, total: 5}}
```

Comment:
Unsure about the `Step 5 of 5`. That didn't appear in the Figma file; should I remove it when displaying the `Let's chat!` title?

Reply:
I think it makes sense to keep it to remain consistent with the previous steps?

Reply:
the step counter hit staging yesterday https://github.com/Expensify/App/pull/5255